src/containers/pages/IRS/Map/helpers.ts

```diff
@@ -0,0 +1,221 @@
+        },
+      },
+      visible: true,
+    };
+    structuresLayers.push(structuresFillLayer);
+
+    const structuresLineLayer = {
+      ...lineLayerConfig,
+      id: `${STRUCTURE_LAYER}-line`,
+      paint: {
+        'line-color': structureStatusColors,
+        'line-opacity': 1,
+        'line-width': 2,
+      },
+      source: {
+        ...lineLayerConfig.source,
+        data: {
+          ...lineLayerConfig.source.data,
+          data: JSON.stringify(structures),
+        },
+      },
+    };
+    structuresLayers.push(structuresLineLayer);
+  }
+  return structuresLayers;
+};
+
+/** Get Gisida Wrapper Props
+ * @param {Jurisdiction} jurisdiction - the jurisdiction (with geojson field)
+ * @param {StructureFeatureCollection} structures - Feature Collection of structures
+ * @param {string[][]} indicatorStops - the indicator stops
+ */
+export const getGisidaWrapperProps = (
+  jurisdiction: Jurisdiction,
+  structures: StructureFeatureCollection | null,
+  indicatorStops: string[][] = defaultIndicatorStops
+): GisidaProps | null => {
+  if (!jurisdiction.geojson) {
+    return null;
+  }
+  const layers: FlexObject[] = [];
+
+  // define line layer for Jurisdiction outline
+  const jurisdictionLineLayer = {
+    ...lineLayerConfig,
+    id: `${MAIN_PLAN}-${jurisdiction.jurisdiction_id}`,
+    source: {
+      ...lineLayerConfig.source,
+      data: {
+        ...lineLayerConfig.source.data,
+        data: JSON.stringify(jurisdiction.geojson),
+      },
+    },
+    visible: true,
+  };
+  layers.push(jurisdictionLineLayer);
+
+  // Define structures layers
+  let structuresLayers: FlexObject[] = [];
+  if (structures) {
+    structuresLayers = structuresLayerBuilder(structures, indicatorStops);
+  }
+
+  // define feature collection of all geoms being rendered
+  const featureCollection =
+    !structures || !structures.features || !structures.features.length
+      ? { features: [jurisdiction.geojson], type: 'FeatureCollection' }
+      : {
+          ...structures,
+          features: [...structures.features, jurisdiction.geojson],
+        };
+  // define bounds for gisida map position
+  const bounds = GeojsonExtent(featureCollection);
+
+  for (const structureLayer of structuresLayers) {
+    layers.push(structureLayer);
+  }
+
+  const gisidaWrapperProps: GisidaProps = {
+    bounds,
+    currentGoal: null,
+    geoData: null,
+    goal: null,
+    handlers: [],
+    layers,
+    pointFeatureCollection: null,
+    polygonFeatureCollection: null,
+    structures: null,
+  };
+  return gisidaWrapperProps;
+};
+
+/** Get breadcrumbs for a jurisdiction object
+ * This uses the jurisdiction_name_path and jurisdiction_path fields to get
+ * the breadcrumbs of the parents of a given jurisdiction object
+ * @param {GenericJurisdiction} jurisdiction - the jurisdiction in question
+ * @param {string} urlPath - the url path that we append the jurisdiction id to
+ */
+export const getJurisdictionBreadcrumbs = (
```

Comment:
We are using the `jurisdiction_name_path` and `jurisdiction_path` properties to get breadcrumbs.